samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -270,7 +270,7 @@ DEFINE_uint32(nireq, 0, infer_requests_count_message);
 DEFINE_uint32(nthreads, 0, infer_num_threads_message);
 
 /// @brief Number of streams to use for inference on the CPU (also affects Hetero cases)
-DEFINE_string(nstreams, "", infer_num_streams_message);
+DEFINE_string(nstreams, "AUTO", infer_num_streams_message);
```

Comment:
BTW, why we need it? AUTO is default in code here https://github.com/openvinotoolkit/openvino/blob/3848a9ec1497c4e3974c8b665ec2105e26406158/samples/cpp/benchmark_app/main.cpp#L305